docs/tutorial/message-ports.md

```diff
@@ -180,19 +180,15 @@ app.whenReady().then(async () => {
 
   // We can't use ipcMain.handle() here, because the reply needs to transfer a
   // MessagePort.
-  ipcMain.on('request-worker-channel', (event) => {
-    // For security reasons, let's make sure only the frames we expect can
-    // access the worker.
-    if (event.senderFrame === mainWindow.webContents.mainFrame) {
-      // Create a new channel ...
-      const { port1, port2 } = new MessageChannelMain()
-      // ... send one end to the worker ...
-      worker.webContents.postMessage('new-client', null, [port1])
-      // ... and the other end to the main window.
-      event.senderFrame.postMessage('provide-worker-channel', null, [port2])
-      // Now the main window and the worker can communicate with each other
-      // without going through the main process!
-    }
+  mainWindow.webContents.mainFrame.on('request-worker-channel', (event) => {
```

Comment:
There's a comment above this line which I feel might be unnecessary. We could replace it to explain why `webContents.mainFrame.on()` is being used.
```suggestion
  // Listen for MessagePort sent from the top-level frame
  mainWindow.webContents.mainFrame.on('request-worker-channel', (event) => {
```